doc/source/cluster/kubernetes/examples/mnist-training-example.md

```diff
@@ -0,0 +1,120 @@
+(kuberay-mnist-training-example)=
+
+# Train a PyTorch Model on Fashion MNIST with CPUs on Kubernetes
+
+This example runs distributed training of a PyTorch model on Fashion MNIST with Ray Train. See [Train a PyTorch Model on Fashion MNIST](../../../train/examples/pytorch/torch_fashion_mnist_example.rst) for more details.
```

Comment:
`../../../train/examples/pytorch/torch_fashion_mnist_example.rst` => We seldom use relative paths. Would you mind adding a label to `torch_fashion_mnist_example.rst` and referring to the document via the label?